src/Controller/FightController.php

```diff
@@ -36,6 +37,11 @@ public function initiate()
     }
     public function statusFight()
     {
+        // choose fighters
+        if (!isset($_SESSION['player1']) || !isset($_SESSION['player2'])) {
+            $fightersController = new FightersController();
+            return $fightersController->pick();
```

Comment:
pick may as well be a method of this controller. then you can go with:

    return $this->pick();

Reply:
FightController and FighterController not same files

Reply:
I didn't confirm my edition. the comment should be:

>pick may as well be in this controller. then you can go with:
>
>     return $this->pick();

Reply:
oh mb

Reply:
Done